src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -37,7 +37,10 @@ file that was distributed with this source code.
     <div class="col-xs-12 col-md-12">
         {% set batchactions = admin.batchactions %}
         {% if admin.hasRoute('batch') and batchactions|length %}
-            <form action="{{ admin.generateUrl('batch', {'filter': admin.filterParameters}) }}" method="POST" >
+            {{ form_start(form, {
+                'method': 'POST',
+                'action': admin.generateUrl('batch', {'filter': admin.filterParameters})
+            }) }}
             <input type="hidden" name="_sonata_csrf_token" value="{{ csrf_token }}">
```

Comment:
Should not we also try to replace this input?

Reply:
Same here: https://github.com/sonata-project/SonataAdminBundle/pull/7727/files#r822988289